specs/0113-light-theme.md

```diff
@@ -0,0 +1,43 @@
+- Feature Name: Light theme
+- Created: 2023-06-22
+- Associated Issue: #2141
+
+# Summary
+
+Implement light theme and enable users to switch between the (current) dark theme and the light theme.
+
+# Motivation
+
+Over the years, many users have expressed preference for a light color theme over the Pontoon default dark theme. Having the ability to switch themes will also allow us to implement high contrast themes, which benefit low vision users, increase readability and reduce website’s visual noise.
+
+# Implementation notes
+
+In the first step, we'll replace existing hardcoded color values with CSS variables. We'll store those in separate CSS files, with one version of each file for each theme.
+
+Users will be able to switch between themes in their settings. Each theme is represented with a title and a radio box. As soon as the theme is selected, it will get applied and the settings will be saved. A "Sync with system theme" checkbox allows users to let the browser pick the theme from the OS setting using the [prefers-color-scheme](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme) CSS media feature.
```

Comment:
I don't think 2 radio buttons plus a checkbox is very intuitive. What's the concern about using 3 radio buttons? See Firefox settings for an example.

Reply:
No particular concern. I like how Slack (and to some extent GitHub) show which theme is selected by the system.

Reply:
Let's see what Eemeli thinks, I'm OK with either.

For reference: Slack, GitHub, Jira, Firefox.

![slack](https://github.com/mozilla/pontoon/assets/3644868/3ff8e6d6-645d-4172-b32c-f62af3b458b2)
![github](https://github.com/mozilla/pontoon/assets/3644868/f051796b-ad00-4e4b-8d2a-2905e64b06f5)
![jira](https://github.com/mozilla/pontoon/assets/3644868/6ce5c72a-21f0-454f-81b0-2ddd2006c0ff)
![firefox](https://github.com/mozilla/pontoon/assets/3644868/26c6ae7b-0236-4a3b-8fc6-c613ea81a426)

Reply:
Note: either way (3rd radio button, or checkbox), I think the specs should say which settings are the default.

To me, match the system sounds like a good default.

Reply:
Matching the system would surprise existing users that use the light theme in their OS, which is most users.

Another argument for the dark default theme is that all our docs use the dark theme.

Reply:
Data shows that 100% of our users use the dark theme ATM. 😛😛😛

Reply:
I have no strong opinion on the shape of the toggle, though it would be nice if it fit in the hamburger menu.